README.md

```diff
@@ -124,52 +131,53 @@ Find more examples in the [examples dir](examples).
 The default jnlp agent image used can be customized by adding it to the template
 
 ```groovy
-containerTemplate(name: 'jnlp', image: 'jenkins/inbound-agent:4.3-4-alpine', args: '${computer.jnlpmac} ${computer.name}'),
+containerTemplate(name: 'jnlp', image: 'jenkins/inbound-agent:4.7-1', args: '${computer.jnlpmac} ${computer.name}'),
 ```
 
 or with the yaml syntax
 
-```
+```yaml
 apiVersion: v1
 kind: Pod
 spec:
   containers:
   - name: jnlp
-    image: 'jenkins/inbound-agent:4.3-4-alpine'
+    image: 'jenkins/inbound-agent:4.7-1'
     args: ['\$(JENKINS_SECRET)', '\$(JENKINS_NAME)']
 ```
 
-### Container Group Support
+### Multiple containers support
 
-Multiple containers can be defined for the agent pod, with shared resources, like mounts. Ports in each container can be accessed as in any Kubernetes pod, by using `localhost`.
+Multiple containers can be defined for the agent pod, with shared resources, like mounts. Ports in each container can
+be accessed as in any Kubernetes pod, by using `localhost`.
 
-The `container` statement allows to execute commands directly into each container.
+The `container` step allows executing commands directly into each container.
```

Comment:
IIRC there was some caveat on this (related to filesystem users and groups on the different containers), don't remember exactly what. Might be worth mentioning (if you do remember better than me).

Reply:
fe10607